SETUP.md

```diff
@@ -3,44 +3,44 @@
 
 ## Before the workshop
 
-* [ ] Have USB drives ready with Anaconda installers and workbook materials (suggested for 2019 workshop)
+* [ ] Have USB drives ready with Miniconda installers and workbook materials.
```

Comment:
```suggestion
* [ ] Have USB drives ready with installers (e.g. anaconda or miniconda) and workbook materials.
```

Since it's not clear whether this is what we'll be using in the future... and anyway we might need a custom-environment one like has been done in the past.